### ISSUE TITLE
Leverage DAO DAO to gather community and make pre GovNo decisions

### ISSUE THREAD

**Issue by CLHRAE** [atomone-hub/genesis#88]
My takeaway from last night's spaces was that there are plenty of decisions that need to be made before launching the GovNo chain.   DAO DAO is a super powerful tool that we can use to gather No / No with Veto voters to start making those decisions. 

In future if deemed helpful it can be used to hold a community fund, and we can leverage subDAOs for things like grants, retroactive compensation etc.

I recorded this DAO DAO demo in order to give an overview on how the DAO can be configured, as well as a potential solution for ensuring that DAO membership is only comprised of No or No with Veto voters (abstain and non voters can come too). 

I'd like to use this issue to discuss DAO configuration, and address any concerns the community might have.  Also open to other solutions. 

Let's dig in...

Here is a link to the demo - https://share.descript.com/view/2OrmXbo1DfD?transcript=false

**Comment by albttx** [comment]
I'm down to help ! :) I'm a big fan of DAODAO

I have the export of the block pre-tally, we can gather all no-voters address, and convert them to juno.

Just how do we gonna organize people to vote from the hub ? I know Juno allow tx with $ATOM, but we might need a custom frontend or something to help people easily bridge and vote no ?

**Comment by giunatale** [comment]
How do we address this fair comment here?

https://twitter.com/0xikaruz/status/1735371296085594532?t=xzhvFZXZMZCp4AaWTzp0dw

It's a compelling argument for why this might be not suitable after all..

**Comment by CLHRAE** [comment]
I responded to the tweet.   DAO DAO recently moved from cw20s to token factory, which as I understand can be made non transferable.

**Comment by ABCharlieEth** [comment]
Have we decided the fate for validators who weighted vote? Also, I completely agree with the usability of DAO DAO.

**Comment by MichaelFrazzy** [comment]
Fantastic, thanks for creating this issue! A non-transferable token for the purposes of GovNo sounds really efficient to me. 

I can't think of any reasons why that would complicate the eventual AtomOne token release versus creating something for GovNo before that from scratch. Either in terms of any type of implementation or weighting the future final airdrop after this one, anything come to mind on anyone else's end?

**Comment by MichaelFrazzy** [comment]
> 
> I have the export of the block pre-tally, we can gather all no-voters address, and convert them to juno.
> 
> 

Oh awesome, could you please send me this when you get a moment? Partially so I can build the airdrop simulation (I'll make sure the parameters are easy to change as decisions are made) and just analytics for fun haha

**Comment by Ticojohnny** [comment]
@MichaelFrazzy @CLHRAE 

I think DAODAO could be a decent interim solution for coordinating GovNo DAO and I had previously suggested it for use with the Decentralists over a year ago but unfortunately ignored.

However if I think if we are as imminent as I'm led to believe... at this point using DAODAO is honestly the "scenic route", and we are late. We should be building a chain to establish validator relations and to sort of signal who is in actuality ready with validator infrastructure to actually want to be a part of this movement.

The only drawback might be that we can't find consensus on what the distribution would be for for the GovNo DAO and that's literally the only advantage I can see to using DAODAO before an actual chain to use for voting. GovNo voting DAO.

I'm under the impression that we should scaffold and fork Atom using Ignite and move forward with GovNo so that's more of a tangible experience than paying for governance proposals using $juno tokens. This aspect seems very illogical, to pay for AtomOne governance using Juno.

To summarize:

1. AtomOne is very imminent
2. We should be building validator relations and realize who actually wants to participate in AtomOne with GovNo as a test trial.
3. We should not be paying for AtomOne governance decisions using Juno tokens.. that's just awkward and far past the concept.

Happy to hear your thoughts.

**Comment by giunatale** [comment]
> Oh awesome @albttx, could you please send me the JSONs or whatever you have for this when you get a moment? Partially so I can build the airdrop simulation (I'll make sure the parameters are easy to change as decisions are made) and just analytics for fun haha 

#65 adds the snapshot but it would require votes also from the subsequent block to be completely accurate. (technically, it would require to apply all state transitions from the next block, https://github.com/atomone-hub/genesis/pull/65#issuecomment-1850967205)

https://github.com/atomone-hub/genesis/blob/main/snapshots/README.md 

props to @albttx who is also working to get a snapshot of prop 82 (for potential future use)

**Comment by MichaelFrazzy** [comment]
@Ticojohnny I can definitely get on board with that thinking too. At least some work would ideally be happening on the chain itself if the DAO DAO route is chosen. 

Does anything feel like we need voting as urgently as possible? If not we could stick to the new chain and simply prepare the weighting/proposal data for that instead of transferring it all to DAO DAO ASAP. Considering the basic voting-only functionality, hopefully not a ton if any work would need redone after GovNo when things are ready for the final token.

**Comment by CLHRAE** [comment]
> @MichaelFrazzy @CLHRAE
> 
> I think DAODAO could be a decent interim solution for coordinating GovNo DAO and I had previously suggested it for use with the Decentralists over a year ago but unfortunately ignored.
> 
> However if I think if we are as imminent as I'm led to believe... at this point using DAODAO is honestly the "scenic route", and we are late. We should be building a chain to establish validator relations and to sort of signal who is in actuality ready with validator infrastructure to actually want to be a part of this movement.
> 
> The only drawback might be that we can't find consensus on what the distribution would be for for the GovNo DAO and that's literally the only advantage I can see to using DAODAO before an actual chain to use for voting. GovNo voting DAO.
> 
> I'm under the impression that we should scaffold and fork Atom using Ignite and move forward with GovNo so that's more of a tangible experience than paying for governance proposals using $juno tokens. This aspect seems very illogical, to pay for AtomOne governance using Juno.
> 
> To summarize:
> 
> 1. AtomOne is very imminent
> 2. We should be building validator relations and realize who actually wants to participate in AtomOne with GovNo as a test trial.
> 3. We should not be paying for AtomOne governance decisions using Juno tokens.. that's just awkward and far past the concept.
> 
> Happy to hear your thoughts.



Yeah, I agree with you on most points.  

To reiterate, the idea to run a pre- GovNo chain daodao dao was to help form consensus around how the GovNo chain would be configured.

I agree wholeheartedly that validator relations are very important, and the earlier  we start organising the better. 

As far as your comment on JUNO, I also agree with this.  It's weird, but was only used as an example for the demo.  
You can require Atom tokens for a deposit if needed. 
<img width="293" alt="Screenshot 2023-12-17 at 10 54 57" src="https://github.com/atomone-hub/genesis/assets/114192151/24e179df-c28c-431b-8141-b2e17e514d00">


Follow on thought.  If GovNo's sole purpose is to supply a gov token to allow us to further organise, why not create a new GovNo DAO DAO deployment? We can start to consolidate validator interest whilst making decisions around how GovNo is configured.  The tools are there and ready to use. No coding required.

I appreciate that I have a huge bias towards DAO DAO, as I'm heavily involved in the project, but as far as I've seen (please prove me wrong), DAO DAO is the most advanced governance tooling in crypto - bold claim, but I've yet to be corrected.  I don't see the point in re-inventing the wheel when the ecosystem already has a solution.  I hope that the community can see that my favour to DAO DAO comes from a place of wanting to help both GovNo and AtomOne.

**Comment by jaekwon** [comment]
DAODAO isn't a replacement for GovNo, and will not be a dependency of GovNo.
DAODAO is an independent experiment but GovNo is a purpose in itself regardless of other tooling.

**Comment by MichaelFrazzy** [comment]
Makes a ton of sense, even if DAO DAO was being used it's necessitate working on the new chain in tandem anyways. So especially if GovNo voting isn't needed yesterday, we can definitely go straight to setting up the voting on a new chain instead of having any type of intermediary step like DAO DAO. Makes sense that you'd not only like to separate the final chain but GovNo and initial voting itself, so that's what we'll shoot for.
